general/compatibility.md

```diff
@@ -19,13 +19,13 @@ The Pantos ecosystem is composed of:
 
   A node responsible for validating cross-chain transfers on behalf of the users.
 
-The Service Node interacts with the Pantos protocol to initiate cross-chain transfers. In order to do so, the Service Node needs to be compatible with the current version of the Pantos protocol. 
+If not stated otherwise, deployed components (Service and Validator nodes) MUST only interact with the latest deployed available version of the Pantos Protocol. The smart contracts offer an existing function to check the currently available version. 
 
 ## Compatibility matrix
```

Comment:
Why does the matrix only contain the service node and protocol? Each of our components implements a particular protocol version, including the contracts. I think the table should list the contracts, validator node, service node, and client library (each one with their versions) on one side and then the maximum (maybe also the minimum) supported protocol version on the other side.